src/Symfony/Component/Asset/Tests/Context/RequestStackContextTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\Tests\Context;
+
+use Symfony\Component\Asset\Context\RequestStackContext;
+
+class RequestStackContextTest extends \PHPUnit_Framework_TestCase
+{
+    public function testConstructor()
+    {
+        $requestStack = $this->getMockBuilder('Symfony\Component\HttpFoundation\RequestStack')
+            ->getMock();
+        $requestStackContext = new RequestStackContext($requestStack);
+
+        $this->assertInstanceOf('Symfony\Component\Asset\Context\RequestStackContext', $requestStackContext);
```

Comment:
This test is not really useful as all it does is it verifies if PHP works properly. All the other tests already cover this case.

I'd see some value in such a test case if it was verifying if a certain interface is implemented (it would communicate certain design decision).